app/models/user.rb

```diff
@@ -1,20 +1,20 @@
 class User < ApplicationRecord
   has_many :games, dependent: :nullify
-  has_secure_password
-
-  mount_uploader :logo, LogoUploader
-  after_initialize :set_defaults
-
-  VALID_EMAIL_REGEX = /\A([\w+\-]\.?)+@[a-z\d\-]+(\.[a-z]+)*\.[a-z]+\z/i
-  validates :email, presence:   true,
-                    uniqueness: { case_sensitive: false },
-                    format:     VALID_EMAIL_REGEX
-
-  validates :company, presence: true,
-                      uniqueness: { case_sensitive: false }
-
-  validates :password, presence: true,
-                       length: { minimum: 6 }
+  # has_secure_password
```

Comment:
Please update with most recent files and then update pull request. User model has been modified and tested already.